includes/class-wc-payments-upe-checkout.php

```diff
@@ -115,6 +115,10 @@ public function get_payment_fields_js_config() {
 			$payment_fields['checkoutTitle']        = $this->gateway->get_checkout_title();
 			$payment_fields['upePaymentIntentData'] = $this->gateway->get_payment_intent_data_from_session();
 			$payment_fields['upeSetupIntentData']   = $this->gateway->get_setup_intent_data_from_session();
+		} elseif ( WC_Payments_Features::is_upe_deferred_intent_enabled() ) {
+			$payment_fields['checkoutTitle'] = $this->gateway->get_checkout_title();
+			$payment_fields['currency']      = get_woocommerce_currency();
+			$payment_fields['cartTotal']     = number_format( WC()->cart->total, 2, '', '' );
```

Comment:
```suggestion
			$payment_fields['cartTotal']     = WC_Payments_Utils::prepare_amount( WC()->cart->get_total( '' ), get_woocommerce_currency() );
```
Any reason you elected not to use this utility function, like we do over [here](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/payment-methods/class-upe-payment-gateway.php#L330), for example?

Reply:
The `prepare_amount` utility function in `WC_Payments_Utils` was overlooked. Changes applied https://github.com/Automattic/woocommerce-payments/pull/5869/commits/9b34bd2a2f9f52b07c1d039240859a9f8623bc48. Thanks!